save-backend/src/main/kotlin/org/cqfn/save/backend/service/ExecutionService.kt

```diff
@@ -15,4 +18,15 @@ class ExecutionService(private val executionRepository: ExecutionRepository) {
     fun saveExecution(execution: Execution) {
         executionRepository.save(execution)
     }
+
+    /**
+     * @param execution
+     */
+    fun updateExecution(execution: ExecutionUpdateDto) {
+        val databaseExecution = executionRepository.findById(execution.id).get()
```

Comment:
You shouldn't rely on incoming request correctness; therefore you shouldn't call `get()` on optional without aditional checks. If someone tries to update non existing execution, the service should return 4xx error